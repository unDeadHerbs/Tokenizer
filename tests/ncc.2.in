"\u5916\u56fd\u8a9e\u306e\u5b66\u7fd2\u3068\u6559\u6388"
"Language Learning and Teaching"
"\u0418\u0437\u0443\u0447\u0435\u043d\u0438\u0435 \u0438 \u043e\u0431\u0443\u0447\u0435\u043d\u0438\u0435 \u0438\u043d\u043e\u0441\u0442\u0440\u0430\u043d\u043d\u044b\u0445 \u044f\u0437\u044b\u043a\u043e\u0432"
"Tere Daaheng Aneng Karimah"
"\u8a9e\u6587\u6559\u5b78\u30fb\u8bed\u6587\u6559\u5b66"
"Ense\u00f1anza y estudio de idiomas"
"\u0418\u0437\u0443\u0447\u0430\u0432\u0430\u043d\u0435 \u0438 \u041f\u0440\u0435\u043f\u043e\u0434\u0430\u0432\u0430\u043d\u0435 \u043d\u0430 \u0427\u0443\u0436\u0434\u0438 \u0415\u0437\u0438\u0446\u0438"
"\u10e5\u10d0\u10e0\u10d7\u10e3\u10da\u10d8 \u10d4\u10dc\u10d8\u10e1 \u10e8\u10d4\u10e1\u10ec\u10d0\u10d5\u10da\u10d0 \u10d3\u10d0 \u10e1\u10ec\u10d0\u10d5\u10da\u10d4\u10d1\u10d0"
"'l\u00e6\u014b\u0261wid\u0292 'l\u0258r:ni\u014b \u00e6nd 'ti:t\u0283i\u014b"
"Lus kawm thaib qhia"
"Ng\u00f4n Ng\u1eef, S\u1ef1 h\u1ecdc,"
"\u202d\u202b\u05dc\u05dc\u05de\u05d5\u05d3 \u05d5\u05dc\u05dc\u05de\u05d3 \u05d0\u05ea \u05d4\u05e9\u05b5\u05e4\u05d4"
"L'enseignement et l'\u00e9tude des langues"
"\ub9d0\ubc30\uc6b0\uae30\uc640\u2003\uac00\ub974\uce58\uae30"
"Nauka j\u0119zyk\u00f3w obcychi"
"\u0393\u03bb\u03c9\u03c3\u03c3\u03b9\u03ba\u1f75 \u0395\u03ba\u03bc\u1f70\u03b8\u03b7\u03c3\u03b7 \u03ba\u03b1\u03b9 \u0394\u03b9\u03b4\u03b1\u03c3\u03ba\u03b1\u03bb\u1f77\u03b1"
"\u202d\u202b\ufe97\ufeaa\ufead\ufbfe\ufeb2 \ufeed \ufbfe\ufe8e\ufea9\ufb94\ufbff\ufeae\ufbfc \ufeaf\ufe91\ufe8e\ufee5"
"Sprachlernen und -lehren"
"\u202d\u202b\ufe97\ufecc\ufee0\ufee2 \ufeed\ufe97\ufeaa\ufead\ufef3\ufeb2 \ufe8d\ufedf\ufecc\ufeae\ufe91\ufef4\ufe94"
"\u0e40\u0e23\u0e35\u0e22\u0e19\u0e41\u0e25\u0e30\u0e2a\u0e2d\u0e19\u0e20\u0e32\u0e29\u0e32"
