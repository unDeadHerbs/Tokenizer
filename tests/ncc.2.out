| Token Type | Token Value
+------------+-- - - -  -  -
|     STRING | '外国語の学習と教授' at 1:1
|     STRING | 'Language Learning and Teaching' at 2:1
|     STRING | 'Изучение и обучение иностранных языков' at 3:1
|     STRING | 'Tere Daaheng Aneng Karimah' at 4:1
|     STRING | '語文教學・语文教学' at 5:1
|     STRING | 'Enseñanza y estudio de idiomas' at 6:1
|     STRING | 'Изучаване и Преподаване на Чужди Езици' at 7:1
|     STRING | 'ქართული ენის შესწავლა და სწავლება' at 8:1
|     STRING | ''læŋɡwidʒ 'lɘr:niŋ ænd 'ti:tʃiŋ' at 9:1
|     STRING | 'Lus kawm thaib qhia' at 10:1
|     STRING | 'Ngôn Ngữ, Sự học,' at 11:1
|     STRING | '‭‫ללמוד וללמד את השֵפה' at 12:1
|     STRING | 'L'enseignement et l'étude des langues' at 13:1
|     STRING | '말배우기와 가르치기' at 14:1
|     STRING | 'Nauka języków obcychi' at 15:1
|     STRING | 'Γλωσσική Εκμὰθηση και Διδασκαλία' at 16:1
|     STRING | '‭‫ﺗﺪﺭﯾﺲ ﻭ ﯾﺎﺩﮔﯿﺮﯼ ﺯﺑﺎﻥ' at 17:1
|     STRING | 'Sprachlernen und -lehren' at 18:1
|     STRING | '‭‫ﺗﻌﻠﻢ ﻭﺗﺪﺭﻳﺲ ﺍﻟﻌﺮﺑﻴﺔ' at 19:1
|     STRING | 'เรียนและสอนภาษา' at 20:1
+------------+-- - - -  -  -
